src/users/users.controller.ts

```diff
@@ -1,13 +1,73 @@
-import { Body, Controller, Post } from '@nestjs/common';
+import { Response } from 'express';
 
+import { Controller, Param, Post } from '@nestjs/common';
+import { Body, HttpCode, Res } from '@nestjs/common/decorators';
+import {
+  Delete,
+  Get,
+  Put,
+} from '@nestjs/common/decorators/http/request-mapping.decorator';
+import { HttpStatus } from '@nestjs/common/enums';
+
+import { CreateUserDto } from './dto/create-user.dto';
 import { UsersService } from './users.service';
 
 @Controller('users')
 export class UsersController {
   constructor(private readonly usersService: UsersService) {}
 
+  @Post('mockCreateUser')
```

Comment:
This part of the code must not be in here. I think you can delete this and use unit test instead.

Reply:
I think using integration test might be more appropriate in this case. The reason is that you can test database too.